api_server.py

```
@@ -0,0 +1,1193 @@
+#!/usr/bin/env python3
+"""
+HTTP API wrapper for video_ingest_tool to interface with Adobe CEP panel.
+Run this alongside the CEP panel to provide HTTP API endpoints.
+"""
+
+import os
+import sys
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)